### PR TITLE
docs: clarify commit prefixes

### PR DESCRIPTION
**Pull request by rix0rrr**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CONTRIBUTING.md
Comment (corymhall) [inline CONTRIBUTING.md:336]:
```suggestion
    * `chore`: something without directly visible user benefit (does not end up in the CHANGELOG). Typically used for build scripts or config changes.
```

I think "minor" changes leaves too much open to interpretation. I think if there is any functionality change, no matter how "minor", it should not be a chore.

File: CONTRIBUTING.md
Comment (kaizencc) [inline CONTRIBUTING.md:332]: Can we also mention that these requirements are not hard requirements and CDK maintainers have the ability to override via labels if they see fit. 

I agree that people are using chore when they should be using fear to skirt these requirements, it would help to know that there are overrides in place.

File: CONTRIBUTING.md
Comment (rix0rrr) [reply CONTRIBUTING.md:336]: I'm not sure. New enum members for example, or a property that should always have been there but wasn't. I mean I guess technically they are features, but it feels a little like scraping the bottom of the barrel to me to put that up in the CHANGELOG. As in, I would feel ashamed putting it there 🫣

Comment (corymhall) [reply CONTRIBUTING.md:336]: Properties that should have been there like these? I feel like it's better to default to being less ambiguous. Core members can always change it to `chore` before approving if they really don't want it to show up in the CHANGELOG.

https://github.com/aws/aws-cdk/blob/6b4f92f2437c7ff782c88ce23925a04168728d7c/packages/%40aws-cdk/aws-ec2/lib/launch-template.ts#L621-L655

But if you think we should be using `chore` more for minor changes then I guess I'm fine with that.

**Review by corymhall — APPROVED** [review]

**Pull request merged**
